contrib/diurnal/example-diurnal-controller.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    name: diurnalcontroller
+  name: diurnalcontroller
+spec:
+  replicas: 1
+  selector:
+    name: diurnalcontroller
+  template:
+    metadata:
+      labels:
+        name: diurnalcontroller
+    spec:
+      containers:
+        - command: ["diurnal_math", "-ns", "default", "-labels", "name=redis-slave", "-sample", "300", "-countfn", "10+10*sin(t)"]
```

Comment:
I think "hard to shoot yourself in the foot" definitely trumps "interesting" when configuring your web services.  

The actual curves are not simple sine() functions.   Composing an curve that has the desired shape using c library math functions is going to be very error-prone and unreadable (e.g. how to you make the weekends lower or higher just using c library functions?  You can do it, but you have to think about it way to long for it to be safe.)

I'd start with a list of (ISO 8601 time, amplitude) pairs.  An enhancement would be to allow different values for certain days of the week.